### PR TITLE
Full JS Stuff

### PR DESCRIPTION
**Pull request by benmcmahon100**
There is a bit more RegEx and if else stuff. Other challenges can be added later but this is the base working set without @QuincyLarson 's fixes to the copy

### CODE REVIEW

File: seed/challenges/basic-javascript.json [hunk 1]
```diff
@@ -26,152 +55,73 @@
...
+      "dashedName": "waypoint-define-your-first-and-last-name",
+      "difficulty": "9.9802",
       "description": [
-        "Now, use the <code>var</code> keyword to create a <code>variable</code> called <code>myFirstName</code> and set its value to your first name. Then set a variable called <code>myLastName</code> to your last name.",
-        "<code>Variables</code> are used to store values.",
-        "Be sure to use lowercase and uppercase letters properly. JavaScript variables are written in <code>lower camel case</code>. An example of lower camel case is: lowerCamelCase.",
-        "Look at the <code>ourFirstName</code> and <code>ourLastName</code> examples if you get stuck."
+        "",
+        "Programs will almost always have several different variables that are used to keep track of several different pieces of data",
+        "We are now going to go and create two new variables <code> myFirstName </code> and <code> myLastName </code> that are strings",
```
Comment (kotp) [inline seed/challenges/basic-javascript.json]: "We are now going to create two..."

**Comment by BerkeleyTrue** [comment]
@benmcmahon100 "_id" must be "id" or things will break.

**Comment by benmcmahon100** [comment]
Hey @BerkeleyTrue just tried to seed with id instead of _id and the seed is hanging so I'm going to avoid using it for now

**Comment by BerkeleyTrue** [comment]
The seed script hangs?

**Pull request merged**